apps/web/public/static/locales/en/common.json

```diff
@@ -749,6 +749,7 @@
   "you_will_only_view_it_once": "You will not be able to view it again once you close this modal.",
   "copy_to_clipboard": "Copy to clipboard",
   "enabled_after_update": "Enabled after update",
+  "enabled_after_update_description": "The link will work after you Update",
```

Comment:
what does that mean?

Reply:
It means that the generated hash can be copied right away but would only work if the edited changes are saved.